src/planning/plan/planificator.ts

```diff
@@ -137,26 +132,26 @@ export class Planificator {
     });
   }
 
-  static constructSuggestionKey(arc: Arc, userid: string, storageKeyBase?: string): KeyBase {
+  static constructSuggestionKey(arc: Arc, storageKeyBase?: string): KeyBase {
     const arcStorageKey = arc.storageProviderFactory.parseStringAsKey(arc.storageKey);
     const keybase = arc.storageProviderFactory.parseStringAsKey(storageKeyBase || arcStorageKey.base());
-    return keybase.childKeyForSuggestions(userid, arcStorageKey.arcId);
+    return keybase.childKeyForSuggestions(planificatorId, arcStorageKey.arcId);
   }
 
-  static constructSearchKey(arc: Arc, userid: string): KeyBase {
+  static constructSearchKey(arc: Arc): KeyBase {
     const arcStorageKey = arc.storageProviderFactory.parseStringAsKey(arc.storageKey);
     const keybase = arc.storageProviderFactory.parseStringAsKey(arcStorageKey.base());
-    return keybase.childKeyForSearch(userid);
+    return keybase.childKeyForSearch(planificatorId);
```

Comment:
how does it work with firebase, where the search strings were all stored under `/${userid}/`? 
if the `userid` becomes now part of the storageKey base, maybe there is no need for planificatorId at all?
i tried looking at the 0_7 entries in fb, and i see there is `/user/search` under each userid, but i cannot find the code that makes it happen, what am missing?

Reply:
I'm not sure what you are asking.
Originally this path would have been `[path to a shared node]/[userid]/search`.
We jettisoned _shared node_, and made the new path: `[path to users node]/user/search` by forcing `userid` variable to have value `user`.
I changed the name of variable `userid` to `planificatorId`, and then set its value to `plans` (instead of `user`). So now, the path is: `[path to users node]/plans/search`

Reply:
sorry, i must have read it wrong.